.github/actions/setup-testground/action.yml

```diff
@@ -50,6 +53,7 @@ runs:
       shell: bash
 
     - name: Run the daemon or configure the client
+      if: ${{ inputs.start_testground == 'true' }}
```

Comment:
> The existing ping tests use the `run-composition` GH action which depends on this being true.

Do I understand correctly that long term that GH action would go away? Do I also understand correctly that we can not yet remove it here given that that would break CIs in rust-libp2p and go-libp2p?

For the sake of cleaning it up right away, would that breakage of our CI be that bad?

Reply:
> Do I understand correctly that long term that GH action would go away?

I wonder if there's still value in these existing ping tests. We get most of the value from them in the new multidim interop tests. The other value is having a fast test that checks if current master works against the other implementations.

>  Do I also understand correctly that we can not yet remove it here given that that would break CIs in rust-libp2p and go-libp2p?

Yeah, and we wouldn't have anything to replace them.

> For the sake of cleaning it up right away, would that breakage of our CI be that bad?

We don't have a plan right now to replace the existing ping tests. I'd rather focus on replacing it and then update the other repo's CI rather than breaking CI and then forcing us to fix it in a reactive way.

Reply:
> The other value is having a fast test that checks if current master works against the other implementations. 

That would as well happen via the multidimensional tests, no?

> Right now there's still some value in the ping tests that isn't captured by this PR.

What value would they provide long term?

Reply:
> That would as well happen via the multidimensional tests, no?

Long term yes. Right now, no.

> What value would they provide long term?

Long term, we should get rid of them. Right now though, I think the priority is to have the multidim interop tests. Replacing the ping tests imo is lower priority.

What the ping test gives us that multidim doesn't right now:
* CI testing in the impl repo.
* Interop testing with Nim (I haven't spent any time getting Nim into the multidim)

Long term, multidim will do this and we can get rid of the ping tests. I would save that for a future PR rather than add to this one. And I would avoid removing the existing ping tests until we're ready to replace them.